src/Common/src/CoreLib/System/Threading/Tasks/ValueTask.cs

```diff
@@ -33,9 +33,27 @@ namespace System.Threading.Tasks
     /// <summary>Provides an awaitable result of an asynchronous operation.</summary>
     /// <remarks>
     /// <see cref="ValueTask"/>s are meant to be directly awaited.  To do more complicated operations with them, a <see cref="Task"/>
```

Comment:
not related to your change, but usually we don't use plural for APIs because it doesn't localize well. So we'd say something like ValueTask objects